docs/root/version_history/current.rst

```diff
@@ -47,6 +47,7 @@ Minor Behavior Changes
   whether receiving `x-envoy-immediate-health-check-fail` will cause exclusion or not. Thus,
   depending on the Envoy deployment, the feature flag may need to be flipped on both downstream
   and upstream instances, depending on the reason.
+* http: add runtime flag to enable internal redirects for requests with body data.
```

Comment:
generally more like

* http: added support for internal redirects with bodies. This behavior can be disabled temporarily by setting `reloadable flag name here` to false.